azureDatabaseUtils/gradle/deploy.gradle

```diff
@@ -1,12 +1,11 @@
-import java.time.ZonedDateTime
-
 // Deploy config
 jib {
   from {
     // docker:// specifies a local image
     image = "docker://azure-database-utils-base:latest"
   }
   extraDirectories {
+    mkdir jibExtraDirectory
```

Comment:
A contribution to jib added validation of the directory specified here. It fails if the directory does not yet exist.